python/common/python/math.java

```diff
@@ -0,0 +1,41 @@
+package python;
+
+@org.python.Module(
+        __doc__ =
+                "This module provides various functions to math." 
+)
+public class math extends org.python.types.Module {
+    public math() {
+        super();
+    }
+
+    @org.python.Attribute
+    public static org.python.Object __file__ = new org.python.types.Str("python/common/python/math.java");
+    @org.python.Attribute
+    public static org.python.Object __loader__ = org.python.types.NoneType.NONE;  // TODO
+    @org.python.Attribute
+    public static org.python.Object __name__ = new org.python.types.Str("math");
+    @org.python.Attribute
+    public static org.python.Object __package__ = new org.python.types.Str("");
+    @org.python.Attribute
+    public static org.python.Object __spec__ = org.python.types.NoneType.NONE;  // TODO
+
+    // Python has a maximum value of float larger than javas. This is pythons max float
+    public static double pythonMaxFloat = 1.7976931348623157e+308;
+
+    @org.python.Method(
+            __doc__ = "exp",
+            args = {"x"}
+    )
+    public static org.python.Object exp(org.python.Object x)
+        throws org.python.exceptions.OverflowError {
+        // The float type uses double internally, so we can simply use that
+        double xValue = ((org.python.types.Float) x.__float__()).value;
```

Comment:
Please explain why you need to add value as a subfix? It's inherently a value. In my opinion this is a code smell and not a good variable name.

Reply:
Because the input parameter of type `org.python.Object` is already named `x`, we need another name! The since `xValue` is the actual value of the object, I deem it to be fitting!

Reply:
Looking at `time.java`, they use e.g. `seconds` as the name of the argument of method `ctime`. To be consistent you should at least rename `x` to `numeric` or something else, along the lines of code of the rest of the project.

Reply:
I renamed `xValue` to `number` and kept `x` as `x` is the name of the parameter in the official python documentation (see https://docs.python.org/2/library/math.html#math.exp)

Reply:
I would argue that we should follow how they have done in this project. Looking at the python documentation (https://docs.python.org/3.6/library/datetime.html?highlight=ctime#datetime.date.ctime), they doesn't even name the input argument.

`xValue` ->`number` is not as ugly and I can accept it.